src/input.c

```diff
@@ -97,16 +105,22 @@ void input_handle_click(unsigned int button, bool button_down, int mouse_x, int
                         continue;
                 }
 
-                if (act == MOUSE_DO_ACTION || act == MOUSE_CLOSE_CURRENT || act == MOUSE_CONTEXT || act == MOUSE_OPEN_URL) {
+                if (act == MOUSE_DO_ACTION || act == MOUSE_CLOSE_CURRENT || act == MOUSE_CONTEXT || act == MOUSE_OPEN_URL || act== MOUSE_SCROLL_SCRIPT) {
                         struct notification *n = get_notification_at(mouse_y);
-
                         if (n) {
                                 if (act == MOUSE_CLOSE_CURRENT) {
                                         n->marked_for_closure = REASON_USER;
                                 } else if (act == MOUSE_DO_ACTION) {
                                         notification_do_action(n);
                                 } else if (act == MOUSE_OPEN_URL) {
                                         notification_open_url(n);
+                                } else if (act == MOUSE_SCROLL_SCRIPT){
```

Comment:
could you please properly indent this? we leave a space between if and (
also spaces between == and commas. but the content seems right